### ISSUE TITLE
supermatter monitor (Most Engineering programs) crashes TGUI

### ISSUE THREAD

**Issue by DaedalusDock-Services** [DaedalusDock/daedalusdock#1000]
Reporting client info:
```Client Information:
BYOND:515.1635
Key:francinum
```

## Round ID:
209

## Testmerges:
- [Debug reboot failures](https://github.com/DaedalusDock/Gameserver/pull/923)
- [Temporary canRecall fix](https://github.com/DaedalusDock/Gameserver/pull/862)
- [Tgooey Restyle](https://github.com/DaedalusDock/Gameserver/pull/969)
- [Bugs 6.16.24](https://github.com/DaedalusDock/Gameserver/pull/994)
- [Sound tweaks](https://github.com/DaedalusDock/Gameserver/pull/996)
- [Hydroponics change and general QOL](https://github.com/DaedalusDock/Gameserver/pull/998)

## Reproduction:
```
ReferenceError: 'useBackend' is undefined
...
State: {"ckey":"francinum","interface":"NtosSupermatterMonitor","window":{"key":"[0x210176b7]-main","size":null,"fancy":1,"locked":0}}
```

**Comment by jlsnow301** [comment]
you're going to hate this but I can't get this to repro on local at all. Does this happen every time? Do you get anything if you tgui-dev on local?

**Comment by francinum** [comment]
Fixed by #1006
